docs/source/spec/core.rst

```diff
@@ -1507,20 +1513,18 @@ of an operation provide values for the identifiers of a resource.
 
 :dfn:`Instance operations` are formed when all of the identifiers of a resource
 are bound to the input structure of an operation or when a resource has no
-identifiers. The :ref:`read <read-lifecycle>` , :ref:`update <update-lifecycle>`,
-and :ref:`delete <delete-lifecycle>` lifecycle operations are examples
-of instance operations. An operation bound to a resource MUST form a valid
-instance operation if it is not marked with the :ref:`collection-trait`.
+identifiers. The :ref:`put <put-lifecycle>`, :ref:`read <read-lifecycle>`,
+:ref:`update <update-lifecycle>`, and :ref:`delete <delete-lifecycle>`
+lifecycle operations are examples of instance operations. An operation bound
+to a resource using `operations` MUST form a valid instance operation.
 
 .. _collection-operations:
 
 :dfn:`Collection operations` are used when an operation is meant to operate on
 a collection of resources rather than a specific resource. Collection
-operations are formed when an operation bound to a resource is marked with the
-``collectionTrait`` and one or more of the child identifiers of a resource
-are not bound to the input structure of an operation. The
-:ref:`list <list-lifecycle>` lifecycle operation is an example of a collection
-operation.
+operations are formed when an operation is bound to a resource with `collectionOperations`,
+or when bound to the :ref:`list <list-lifecycle>` or :ref:`create <create-lifecycle>`
+lifecycle operations.
```

Comment:
I think this needs to define how to create a valid collection operation. You have to omit one or more identifiers of the resource it's bound to (though all parent identifiers must be bound to the input of the operation).